sdks/python/apache_beam/testing/load_tests/sideinput_test.py

```diff
@@ -122,85 +122,95 @@ def process(self, unused_element, side_input):
             # No-op. We only make sure that the element is accessed.
             next(it)
           except StopIteration:
-            return
+            break
 
     class MappingSideInputTestDoFn(beam.DoFn):
```

Comment:
Ok.
Since we don't support Python 2 anymore, I decided to use Python 3 annotations which are slightly better than class decorators (@typehints.with_input_types and @typehints.with_output_types)